src/workerd/jsg/jsg.h

```diff
@@ -2014,6 +2016,33 @@ class Lock {
   // ---------------------------------------------------------------------------
   // Misc. Stuff
 
+  v8::Local<v8::Value> call(v8::Local<v8::Function> fn,
+                            v8::Local<v8::Value> thisArg = v8::Local<v8::Value>()) {
+    if (thisArg.IsEmpty()) thisArg = v8Isolate->GetCurrentContext()->Global();
+    return call(fn, thisArg, kj::Array<v8::Local<v8::Value>>());
+  }
+
+  template <typename... Args>
+  v8::Local<v8::Value> call(v8::Local<v8::Function> fn, v8::Local<Args>... args) {
+    return call(fn, v8Isolate->GetCurrentContext()->Global(), kj::fwd<Args...>(args...));
```

Comment:
Shouldn't `kj::fwd<Args...>(args...)` be a compile error, since `args` are `v8::Local<Args>...`, not `Args&&...`? Just passing `args...` might make the most sense, since I think `v8::Local`s are trivially copyable.

I assume this means that this overload is never being called, which I think makes sense: given
```cpp
v8::Local<v8::Function> fn;
v8::Local<v8::Value> arg1;
v8::Local<v8::Value> arg2;
```
I'd expect the expression `call(fn, arg1)` to match the top-most overload, either because it's not a template or because it's most specific (I don't remember the exact rules). This is despite both of the other two inline overloads having matching signatures.

Meanwhile, I'd expect the expression `call(fn, arg1, arg2)` to match the bottom-most overload, because it's more specific than this one. Thus, I think if the second argument is a `v8::Local<v8::Value>`, it will always be interpreted as the `this` argument.

It looks like the only parts of the codebase which relies on the "`this` is the global context by default" behavior happen to call the ArrayPtr non-inline overload instead, so I think we can just delete this overload.

If there's an easy test or two to write for this API, that'd be a good idea.

Reply:
I think this `call(...)` variation isn't actually called anywhere right now so I'll just remove it.